pkg/pool-stable-phantom/test/StablePhantomPool.test.ts

```diff
@@ -85,6 +85,11 @@ describe('StablePhantomPool', () => {
       expect(await pool.inRecoveryMode()).to.be.true;
     }
 
+    async function disableRecoveryMode(pool: StablePhantomPool): Promise<void> {
+      await pool.disableRecoveryMode(admin);
+      expect(await pool.inRecoveryMode()).to.be.false;
+    }
```

Comment:
These helpers are ok, but feel free to just call `disableRecoveryMode` and not check the result afterwards - we don't really need that test.